src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -385,7 +359,7 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
     return this.queryStateModel.get(this.queryStateAttribute) || this.options.basePath;
   }
 
-  public get queryStateAttribute() {
```

Comment:
Wouldn't changing `public` to `private` be breaking?

Reply:
We decided that we wouldn't break public *documented* methods, as they are the one that can be seen in the docs. 
Should I also leave every public variable & method public? For queryStateAttribute sure but for others... some have changed or have no use anymore. IMO way too many methods we're made public in that original implementation.